fsb/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+#![allow(non_snake_case)]
+#[allow(dead_code)]
+#[macro_use]
+mod macros;
+mod pi;
```

Comment:
GitHub won't even let me leave comments on that module, but... wow 😮

Reply:
Not my work! Copied that file from the reference implementation